kubernetes/loculus/values.yaml

```diff
@@ -1234,13 +1234,15 @@ defaultOrganisms:
         version: 1
         configFile:
           <<: *preprocessingConfigFile
+          nextclade_sort: True
```

Comment:
The name might not be obvious as to what happens here. Let's use something like `require_nextclade_sort_match`?